.github/workflows/publish-python-wrapper-client.yaml

```diff
@@ -32,4 +33,34 @@ jobs:
         packages-dir: clients/python-wrapper/dist/
         verbose: true
 
-# TODO: Publish docs once we decide on documentation process
+    - name: Extract docs version
+      working-directory: ./clients/python-wrapper/
+      shell: bash
+      run: |
+          echo "tag=$(python setup.py --version)" >> $GITHUB_OUTPUT
+      id: docver
+
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v4
+      with:
+        python-version: 3.9
+
+    - name: Generate and Build Documentation
+      working-directory: ./clients/python-wrapper/
+      run: |
+        echo ${{ steps.docver.outputs.tag }}
+        pip install -r requirements.txt 
+        sphinx-apidoc -o docs lakefs sphinx-apidoc --full -A 'Treeverse' -e
+        sphinx-build -b html docs docs/_build/${{ steps.docver.outputs.tag }}
+        cp README.md docs/_build 
+
+    - name: Publish to Docs Repository
+      uses: dmnemec/copy_file_to_another_repo_action@v1.1.1
+      env:
+        API_TOKEN_GITHUB: ${{ secrets.PERSONAL_TOKEN }}
+      with:
+        source_file: clients/python-wrapper/docs/_build/.
+        destination_repo: treeverse/docs-lakefs-sdk-python-wrapper
+        destination_folder: /
```

Comment:
verify that this action add files and not replace, when we publish new version we can copy the specific one to the right destination in case it will try to remove the rest of the files found in destination folder.

Reply:
It's just a cp operation - it's the same action we used for the python sdk as well